### DISCUSSION TITLE
Can you create Plugins Manager Tool for auto check/update/Add/Remove for QuickLook?

### DISCUSSION THREAD

**Discussion by khoighost** [QL-Win/QuickLook#1341] [Ideas]
Can you create Plugins Manager Tool for auto Check/Update/Add/Remove for QuickLook?

Thanks! :D

**Comment by rabelux** [comment]
There has been an attempt of solving this in #415 but somehow it didn't get finished. So I guess you shouldn't expect a solution in the next months/years 😢
